spec/index.bs

```diff
@@ -11607,6 +11677,72 @@ It must only be included by interfaces which also include those mixins.
     Implementations are strongly encouraged to optimize this algorithm.
 </div>
 
+## Immediate Data ## {#programmable-passes-immediate-data}
+
+<dl dfn-type=method dfn-for=GPUBindingCommandsMixin>
+    : <dfn>setImmediates(rangeOffset, data, dataOffset, size)</dfn>
+    ::
+        Sets immediate data for subsequent render or compute commands.
+
+        <div algorithm=GPUBindingCommandsMixin.setImmediates>
+            <div data-timeline=content>
+                **Called on:** {{GPUBindingCommandsMixin}} |this|.
+
+                **Arguments:**
+
+                <pre class=argumentdef for="GPUBindingCommandsMixin/setImmediates(rangeOffset, data, dataOffset, size)">
+                    |rangeOffset|: Offset in bytes into the immediate data range to begin writing at.
+                        Must be a multiple of 4.
+                    |data|: Data to write into the immediate data range.
+                    |dataOffset|: Offset into |data| to begin writing from. Given in elements if
+                        |data| is a {{TypedArray}} and bytes otherwise. Defaults to 0.
+                    |size|: Size of content to write from |data|. Given in elements if
```

Comment:
nit: Maybe this should be called `dataSize`, so it's clearly associated with the type of `data` (not specified in bytes). We can change `writeBuffer` as well.